### PR TITLE
Convert UI automation code to use Typescript

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: ui-automated-tests/src/flows/brief/atm.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  await clickSaveContinue();
+  await utils.matchText("li", "You must select who can respond");
+  await utils.selectRadio("category");
+  await clickSaveContinue();
+  await utils.matchText("li", "You must select a panel category");
+  await utils.selectRadio("all");
+  await clickSaveContinue();
+};
+
+const fillAbout = async (title: string, locations: string[]) => {
```
Comment (resh-ie) [inline ui-automated-tests/src/flows/brief/atm.ts:40]: maybe add some code to check for the validation error messages?
Enter the title for your brief
Enter the name of your organisation
Enter a summary of your brief
You must select at least one location

File: ui-automated-tests/src/flows/brief/atm.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  await utils.type("workAlreadyDone", { numberOfWords: 500 });
+  await utils.type("industryBriefing", { numberOfWords: 500 });
+  await utils.upload("file_0", "document.pdf");
+  await clickSaveContinue();
+};
+
+const fillTimeframes = async () => {
+  await clickSaveContinue();
+  await utils.matchText("li", "Enter an estimated start date for the brief");
+
```
Comment (resh-ie) [inline ui-automated-tests/src/flows/brief/atm.ts:90]: might need to run npm prettier
Remove the unneeded space.

File: ui-automated-tests/src/flows/brief/atm.ts (hunk 1 above)
Comment (malmas18) [reply ui-automated-tests/src/flows/brief/atm.ts:40]: added

File: ui-automated-tests/src/flows/brief/atm.ts (hunk 2 above)
Comment (malmas18) [reply ui-automated-tests/src/flows/brief/atm.ts:90]: linting will not pick this one up.  it is common to have a line to show sections in code

**Review by resh-ie — APPROVED** [review]

**Pull request merged**